api/tests/test_asset_log.py

```diff
@@ -43,6 +43,23 @@ def test_add_checkin(self):
             log_type="Checkin",
         )
         self.assertEqual(AssetLog.objects.count(), 3)
+        created_log = AssetLog.objects.filter(asset=self.test_other_asset).first()
+        self.assertEqual(created_log.log_type, "Checkin")
+
+    def test_verify_double_checkin_for_asset(self):
+        # First log
+        AssetLog.objects.create(
+            checked_by=self.security_user,
+            asset=self.test_other_asset,
+            log_type="Checkin",
+        )
+        # Second log
+        AssetLog.objects.create(
+            checked_by=self.security_user,
+            asset=self.test_other_asset,
+            log_type="Checkin",
+        )
+        self.assertEqual(AssetLog.objects.count(), 3)
```

Comment:
It's not clear why the logs count is 3. a better way is to check that the 2nd log errors out. (see comment [here](https://github.com/AndelaOSP/art-backend/pull/265/files#diff-bea1207c116ad66b7b398915447bd091R364))